src/test/resources/checks/v3/format/OAR113/invalid.yaml

```diff
@@ -0,0 +1,26 @@
+openapi: "3.0.0"
+info: # Noncompliant {{OAR113: Field or extension x-custom-example must be at the assigned location}}
+  version: "1.0.0"
+  title: "Sample API"
+paths:
+  /example:
+    get:
+      description: "Get example"
+      responses:
+        "200":
+          description: "Successful response"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ExampleObject"
+components:
+  schemas:
+    ExampleObject:
+      type: object
+      properties:
+        id:
+          type: string
+        name:
+          type: string
+      x-extension-field: "example"
```

Comment:
The not valid example should check the missing field in each of the nodes supported by the rule present in the test file